### PR TITLE
move rest/jersey/console classes from scalate-core to scalate-jersey …

### PR DESCRIPTION
**Pull request by maslovalex**
see #72

### CODE REVIEW

**Comment by rkrzewski** [comment]
Thanks, I'll review it later tonight.

File: pom.xml [hunk 1]
```diff
@@ -70,8 +70,8 @@
 
     <osgi-version>4.2.0</osgi-version>
 
-    <scala-version>2.11.0</scala-version>
     <scala-version-suffix>2.11</scala-version-suffix>
+    <scala-version>${scala-version-suffix}.0</scala-version>
```
Comment (rkrzewski) [inline pom.xml]: Is this change necessary? I think it does not change semantics down the road.

**Comment by rkrzewski** [comment]
Overall, the change looks good. I was about to ask you to update the sbt build too, but then I realized that it's really ancient: it's using sbt 0.7.4 and Scala 2.7.7! So it's a subject for another PR :)

File: pom.xml (hunk 1 above)
Comment (maslovalex) [inline pom.xml]: not really, it's just how I prefer it to be (less changes when scala version changed).

Comment (rkrzewski) [inline pom.xml]: fair enough.

**Comment by rkrzewski** [comment]
Done. Thanks for your contribution, @maslovalex
